modules/ob-creating-a-buildah-build.adoc

```diff
@@ -0,0 +1,111 @@
+// This module is included in the following assembly:
+//
+// * work-with-builds/using-builds.adoc
+
+:_content-type: PROCEDURE
+[id="ob-creating-a-buildah-build_{context}"]
+= Creating a buildah build
+
+You can create a `buildah` build and push the created image to the {ocp-product-title} cluster internal registry.
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `buildah` strategy. You can use the following command to verify if the `buildah` strategy is installed:
++ 
+[source,terminal]
+----
+$ oc get cbs | grep buildah
+----
++
+[NOTE]
+====
+If the `buildah` strategy is not installed, see _Installing buildah strategy_ in the _Additional resources_ section.
+====
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: buildah-golang-build
+spec:
+  source: <1>
+    git: 
+      url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  strategy: <2>
+    name: buildah
+    kind: ClusterBuildStrategy
+  paramValues: <3>
+  - name: dockerfile
+    value: Dockerfile
+  output: <4>
+    image: image-registry.openshift-image-registry.svc:5000/buildah-example/sample-go-app
+EOF
+----
+<1> The location where the source code is placed.
+<2> The build strategy that you use to build the container.
+<3> A name-value list to specify values for parameters defined in the build strategy. To set the value of the `dockerfile` strategy parameter, specify the Dockerfile location required to build the output image.
+<4> The location where the built image is pushed. By default, the built image is pushed to the {ocp-product-title} cluster internal registry. `buildah-example` is the name of the current project. Ensure that the specified project exists to allow the image push.
+
```

Comment:
We need a step here to check the build
 To view the *Build* which you just created:
oc get builds.shipwright.io
NAME                   REGISTERED   REASON      BUILDSTRATEGYKIND      BUILDSTRATEGYNAME   CREATIONTIME
buildah-golang-build   True         Succeeded   ClusterBuildStrategy   buildah             16s

And explanation for the output
@apoorvajagtap can you please give these